magenta-lib/src/main/scala/magenta/input/DeploymentResolver.scala

```diff
@@ -0,0 +1,78 @@
+package magenta.input
+
+
+object DeploymentResolver {
+  val DEFAULT_REGIONS = List("eu-west-1")
+
+  def resolve(config: RiffRaffDeployConfig): List[Either[(String, String), Deployment]] = {
```

Comment:
Using simple built in types is nice though. There's definitely a point where this becomes complex enough for it to warrant the cost of using a case class everywhere to get named properties. I don't think 'two strings' is quite at that point (although I concede as the co-author my opinion is biased here). I'll give it a try though, see how it looks.